src/Poseidon/GenotypeData.hs

```diff
@@ -25,6 +25,16 @@ import           SequenceFormats.Plink      (readFamFile, readPlink)
 import           System.Console.ANSI        (hClearLine, hSetCursorColumn)
 import           System.FilePath            ((</>))
 import           System.IO                  (hFlush, hPutStr, hPutStrLn, stderr)
+
+-- | A simplified datatype to specify genotype files for ínput purposes
+data InGenotypeData = InGenotypeData
```

Comment:
Isn't this almost 100% code duplication from `GenotypeDataSpec`?

Reply:
Yeah - I started with that, but then ran into issues when writing `parseInGenotypeDataset :: OP.Parser GenotypeDataSpec`, because the input does not have checksums. I then also though it might be conceptually useful to keep the input perspective separate. 

How would I write a parser for `GenotypeDataSpec` that sets the checksums to `Nothing`?

Reply:
Very easy using applicative syntax and `pure Nothing`. I can write a more concrete code snippet later.

Reply:
I think I understand. I read a blog post about applicative yesterday. The `pure` was missing in my first attempt.

Reply:
Right. Like this:

```
parseInGenotypeDataset :: OP.Parser InGenotypeData
parseInGenotypeDataset = InGenotypeData <$> parseInGenotypeFormat
                                        <*> parseInGenoFile
                                        <*> pure Nothing
                                        <*> parseInSnpFile
                                        <*> pure Nothing
                                        <*> parseInIndFile
                                        <*> pure Nothing
                                        <*> parseGenotypeSNPSet
```

Reply:
I removed `InGenotypeData` :+1: